eqdsk/file.py

```diff
@@ -673,7 +712,6 @@ def write_array(fortran_format: ff.FortranRecordWriter, array: np.ndarray):
         qpsi = (
             np.zeros(data["nx"]) if data["qpsi"] is None else np.atleast_1d(data["qpsi"])
         )
-        eqdsk_warn("No qpsi data found. Setting as array of 0's.")
```

Comment:
I think this warning should maybe be kept as 0s are not 1 or -1?

Reply:
Been through it and this is resolved. Tests and code updated.